src/index.js

```diff
@@ -181,6 +181,13 @@ promExporterApp.listen(9599, () => {
               throw "Unauthorized";
             }
 
+            // prevent access if no agency is configured on client
+            if (!request.profile?.agency) {
+              log.error("Missing configuration for token client", request.smaug?.app?.clientId);
+              response.status(400);
+              return response.send({statusCode: 400, message: "Bad request"});
```

Comment:
Synes godt vi kan sige til kalderen er det er en ugyldig client og ikke bare en bad request. For ellers sidder de ret fast